app/views/layouts/application.html.erb

```diff
@@ -6,11 +6,15 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
     
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css">
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %> 
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
   <body>
+    <%= render 'shared/navbar' %>
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
     <%= yield %>
```

Comment:
- Please make available the page(login) where to check if the codes are working: Here is the file where the errors is been noticed. 
- 
![Capture](https://user-images.githubusercontent.com/63972670/201919303-479d2f32-d2f5-4453-8499-7bd6680f1a47.JPG)

Reply:
@OLIPLICHE I am not getting this error locally. Attached are images as proof or evidence
![image](https://user-images.githubusercontent.com/30408871/201921272-53651d99-6c9b-471a-8c46-8c13327bdc37.png)

![image](https://user-images.githubusercontent.com/30408871/201921426-f09e7c4e-2492-40f9-9c5f-fda580c32589.png)

![image](https://user-images.githubusercontent.com/30408871/201921481-301574d3-4a48-472a-a146-11a3907699d2.png)

![image](https://user-images.githubusercontent.com/30408871/201921789-46069d5f-128e-4eba-b0ba-34538bc38d47.png)

Maybe you need to run rspec::webpack:install 

So you can fix that in your local environment

Please check again

Reply:
@OLIPLICHE 
bundle exec rails webpacker:install

Kindly help run the following command to update webpack locally at your end

This is from your end bro.

Reply:
@OLIPLICHE 
Could you please run the following commands
rails webpacker:install
bundle exec rake assets:precompile

Reply:
Hello, I set my environment before running the App. If you sure about what you're are saying, Kindly submit. The second reviewer will check it. Sorry for late answer, I am outside, using my phone to reply.

Reply:
@ogagaoghene Running the commands provided didn't solve the issue, I did use  npm install, am used ubuntu to run the app, my observation `this was caused by your use of bootstrap CDN which required node modules to be setup`

Reply:
@danmainah Thank you for the hint. I have learnt something. I am glad you were able to figure it out. Thanks bro.